### PR TITLE
Nodejs-week1

### PR DESCRIPTION
**Pull request by Hailegebriel22**

### CODE REVIEW

File: nodejs/week1/avg.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+}
+
+else if (array.every(element => !isNaN(element))) {
+    const average = array.reduce((a, b) => a + b) / array.length;
+    console.log(average)
+}
+else {
+    console.log('Provided arguments are not numbers')
+}
+// / (process.argv.length-2)
```
Comment (itsgauravsaxena) [inline nodejs/week1/avg.js:11]: Please remove commented code.

File: nodejs/week1/avg.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+
+const array = process.argv.slice(2).map(num => Number(num));
+if (array.length === 0) {
+    console.log('No arguments are provided')
+}
+
```
Comment (itsgauravsaxena) [inline nodejs/week1/avg.js]: Normally else and elseif should start right where if statement closes. Like
https://stackoverflow.com/a/4005623/4261654

**Review by itsgauravsaxena — APPROVED** [review]

File: nodejs/week1/avg.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+else if (array.every(element => !isNaN(element))) {
+    const average = array.reduce((a, b) => a + b) / array.length;
```
Comment (Hanouj) [inline nodejs/week1/avg.js:6]: you have used arrow functions expression really nicely in this hw

File: nodejs/week1/avg.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+
+const array = process.argv.slice(2).map(num => Number(num));
+if (array.length === 0) {
+    console.log('No arguments are provided')
+}
```
Comment (Hanouj) [inline nodejs/week1/avg.js]: nice work with if and else statements just use less space and put them closer together

**Review by Hanouj — COMMENTED** [review]
Haile overall you did a good job with implementing this warmup

**Pull request merged**